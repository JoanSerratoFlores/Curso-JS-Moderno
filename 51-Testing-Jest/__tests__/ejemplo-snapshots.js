const cliente = {
    nombre: ' Juan 2',
    balances: 500,
    tipo : ' Premium'
};

describe('Testng al cliente', () => {
    test('Es Juan pablo', () => {
        expect(cliente.nombre).toMatchSnapshot();
    });
});