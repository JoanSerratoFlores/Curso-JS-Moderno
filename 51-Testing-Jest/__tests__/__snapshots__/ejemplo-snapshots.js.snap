// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testng al cliente Es Juan pablo 1`] = `" Juan 2"`;
